src/Storages/ObjectStorageQueue/ObjectStorageQueueIFileMetadata.cpp

```diff
@@ -225,9 +225,12 @@ bool ObjectStorageQueueIFileMetadata::setProcessing()
     auto state = file_status->state.load();
     if (state == FileStatus::State::Processing
         || state == FileStatus::State::Processed
-        || (state == FileStatus::State::Failed && file_status->retries >= max_loading_retries))
+        || (state == FileStatus::State::Failed
+            && file_status->retries
```

Comment:
I'm not sure I understand this part, would it be same as checking if `max_loading_retries` is not 0?